data/eos-branch

```diff
@@ -0,0 +1,37 @@
+# Endless product-specific branches configuration
+#
+# Copyright (C) 2016 Endless Mobile Inc.
```

Comment:
Actually, we shouldn't merge this file here, as it is going to live in eos-administration and be deployed by some jenkins+ansible magic (see endlessm/eos-shell#6417). But I'll take your point and remove the copyright notice there.